File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/FFprobeSessionCompleteCallback.java
package com.lgcns.ffmpegkit;

/**
 * <p>Callback function that is invoked when an asynchronous <code>FFprobe</code> session has ended.
 */
@FunctionalInterface
public interface FFprobeSessionCompleteCallback {

    /**
     * <p>Called when an FFprobe session has ended.
     *
     * @param session FFprobe session
     */
    void apply(final FFprobeSession session);

}

File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/FFprobeSessionCompleteCallback.kt
package com.lgcns.ffmpegkit

/**
 *
 * Callback function that is invoked when an asynchronous `FFprobe` session has ended.
 */
fun interface FFprobeSessionCompleteCallback {
    /**
     *
     * Called when an FFprobe session has ended.
     *
     * @param session FFprobe session
     */
    fun apply(session: FFprobeSession?)
}